solr/core/src/test/org/apache/solr/handler/TestReplicationHandlerDiskOverFlow.java

```diff
@@ -114,6 +116,7 @@ public void tearDown() throws Exception {
       followerClient = null;
     }
     System.clearProperty(TEST_URL_ALLOW_LIST);
+    System.clearProperty("solr.security.allow.urls.enabled");
```

Comment:
Clearing properties in an `@After` or `@AfterClass` is always pointless since our test infra does that for us.  Removing them everywhere would be a nice separate PR.

Reply:
Good suggestion, will do that.